tests/baselines/reference/errorOnInitializerInObjectType.errors.txt

```diff
@@ -0,0 +1,17 @@
+tests/cases/compiler/errorOnInitializerInObjectType.ts(2,17): error TS1246: An object type property cannot have an initializer.
+tests/cases/compiler/errorOnInitializerInObjectType.ts(6,17): error TS1246: An object type property cannot have an initializer.
+
+
+==== tests/cases/compiler/errorOnInitializerInObjectType.ts (2 errors) ====
+    interface Foo {
+        bar: number = 5;

```

Comment:
Actually, this is because the parser tries to be smart and avoids reporting more errors at the same start position. I'm not sure if that's the right thing to do because instead of gracefully parsing the invalid initializer, the parser can start going haywire down the line. Maybe `tryParse` would be better? What do you think @CyrusNajmabadi?